imports/plugins/core/orders/client/templates/orders.js

```diff
@@ -298,7 +298,12 @@ Template.orderStatusDetail.helpers({
   orderAge: function () {
     return moment(this.createdAt).fromNow();
   },
-
+  orderStatus: function () {
```

Comment:
some of those functions exists in the app, I used es6 to declare my own functions.